src/main/java/com/faforever/client/replay/ReplayDetailController.java

```diff
@@ -424,6 +438,10 @@ private CompletableFuture<Void> enrichReplayLater(Path path, ReplayBean replay)
   private void populateTeamsContainer(Map<String, List<GamePlayerStatsBean>> newValue) {
     CompletableFuture.supplyAsync(() -> createTeamCardControllers(newValue)).thenAcceptAsync(controllers -> {
       teamCardControllers.clear();
+      if (controllers.stream()
+          .map(TeamCardController::getTeamResult).noneMatch(gameOutcome -> gameOutcome != GameOutcome.DEFEAT)) {
+        controllers.forEach(teamCardController -> teamCardController.setTeamResult(GameOutcome.DRAW));
+      }
```

Comment:
I need to check if both controllers are reporting defeat. So I can't decide this in a single controller. Or do you mean to just move these lines as is to the end of the createTeamCardControllers function?

Reply:
Months ago when I was starting to develop this, I found a game that reported me that both teams were defeated. I didn't save the replay Id, so I can't check now. It might have been because of my own faulty calculations at that time. E.g. only a minority reported a draw, so the majority of defeat results got chosen instead.
There is no way to know now, so I will remove this check and if we ever run into a replay that reports both teams as defeated we can investigate again.